crates/router/src/core/fraud_check/flows/transaction_flow.rs

```diff
@@ -145,6 +146,9 @@ impl
             integrity_check: Ok(()),
             additional_merchant_data: None,
             header_payload,
+            connector_mandate_request_reference_id: Some(common_utils::generate_id_with_len(
```

Comment:
If we generate a random ID / string but don't store it at all in our database, then it would be of no use to us. Can we please avoid such usages?